### ISSUE TITLE
Play a Tween while in Edit Mode

### ISSUE THREAD

**Issue by IvanNSBS** [Demigiant/dotween#471]
I'm trying to create a custom inspector to help my team with our dotween animations. The thing is, i'd like to preview it, in the custom inspector window or even while in edit mode. 

Is there any way to achieve this? Having to be in play mode to test animations slows down development time quite a bit.

**Comment by krisrok** [comment]
yes, it is possible and a truly great feature. have a look at [DOTweenEditorPreview](https://github.com/Demigiant/dotween/blob/a9012224a3d56f829f012d08b03ea72c7f405839/_DOTween.Assembly/DOTweenEditor/DOTweenEditorPreview.cs).
